sympy/matrices/common.py

```diff
@@ -1098,7 +1098,6 @@ def jordan_block(kls, size=None, eigenvalue=None, **kwargs):
             ).warn()
 
         klass = kwargs.pop('cls', kls)
-        band = kwargs.pop('band', 'upper')
         rows = kwargs.pop('rows', None)
         cols = kwargs.pop('cols', None)
```

Comment:
I see why rows and cols are left but shouldn't kls and eigenval be explicit keyword arguments?